docs/03-guidelines/releases/01-release-strategy.md

```diff
@@ -126,7 +126,7 @@ After reaching the end of the development cycle, the Kyma developers create a re
 - two working days for minor releases
 - one week for major releases
 
-All release candidates must always be fully tested by all defined automated and manual tests, following the [test strategy](/guidelines/technical-guidelines/07-test-strategy.md). Manual tests are tracked in [this sheet](https://docs.google.com/spreadsheets/d/1ty3OciQzgzv0GagTG2Dku9os2AfMupbGNf8QxjHaO88/edit) for each release candidate.
+All release candidates must always be fully tested by all defined automated and manual tests, following the [test strategy](../technical/07-test-strategy.md). Manual tests are tracked in [this sheet](https://docs.google.com/spreadsheets/d/1ty3OciQzgzv0GagTG2Dku9os2AfMupbGNf8QxjHaO88/edit) for each release candidate.
 
```

Comment:
It's hard to fix this link, because in website we have different anchor headers and it's not easy to fix in current situation.
I can bypass it by changing the link to: `release-strategy-critical-issues`